core/tests/release_sources/invalid_app_dev.yaml

```diff
@@ -0,0 +1,52 @@
+default_expiration: "10m"
```

Comment:
You should probably have a comment at the top of this file explaining in what way(s) it is invalid.

Reply:
Done.